Automattic/wp-calypso
# Hosting dashboard: remove initial selected site hack

Fixes:

- https://github.com/Automattic/wp-calypso/issues/93827

## Proposed Changes

This PR removed the hack which "clicks" the initial selected site in the hosting dashboard. We can now pass an initial selection to the DataViews component.

## Why are these changes being made?

We have upgraded DataViews package to the latest version which support controlled mode.

## Testing Instructions

1. Go to /sites.
2. Click a site.
3. Verify that the preview pane is open.
4. Verify that the clicked site is in the selected state in the left "sidebar".

## Pre-merge Checklist
